torch/csrc/cuda/Module.cpp

```diff
@@ -118,7 +119,7 @@ PyObject * THCPModule_getRNGState(PyObject *_unused)
   using namespace at;
   using namespace torch::autograd;
   HANDLE_TH_ERRORS
-  Variable var = VariableType::getType(CPU(kByte))->tensor();
+  Variable var = torch::empty({0}, at::device(at::kCPU).dtype(at::kByte));
```

Comment:
Btw `IntList` (or rather `ArrayRef<T>`) has a constructor that takes a single value and constructs a list from it. This way right here frequently causes warnings because the braces can be interpreted as constructing a single value or an initializer list, i.e. both `T` and `initializer_list<T>`. It's better to just use the value without the braces for `ArrayRef` when you're passing a single value